src/platform/imx8/imx8.x.in

```diff
@@ -148,7 +148,8 @@ _memmap_cacheattr_bp_strict = 0x22F22FFF;
 _memmap_cacheattr_wb_allvalid = 0x44224222;
 _memmap_cacheattr_wt_allvalid = 0x11221222;
 _memmap_cacheattr_bp_allvalid = 0x22222222;
-PROVIDE(_memmap_cacheattr_reset = _memmap_cacheattr_wbna_trapnull);
+_memmap_cacheattr_imx_wt_trapnull = 0x22212222;
+PROVIDE(_memmap_cacheattr_reset = _memmap_cacheattr_imx_wt_trapnull);
 
```

Comment:
Lets use imx8_ instead of imx_ in the name here. Other than that, it looks good to me!

Reply:
Will also rename to _allvalid when I come back to it.

Reply:
Indeed :)

Reply:
Done.

Reply:
Done.